tests/extmod/vfs_lfs_error.py

```diff
@@ -1,7 +1,7 @@
 # Test for VfsLittle using a RAM device, testing error handling
 
 try:
-    import vfs
+    import errno, vfs
```

Comment:
Is it necessary to add SKIP logic for ports without `MICROPY_PY_ERRNO` set (defaults to `MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_EXTRA_FEATURES`), or is any port that can run extmod tests likely to have this set already?

Reply:
(`vfs` defaults to `MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_CORE_FEATURES && MICROPY_VFS`, so there is a potential window there but I don't know if there's actually many boards configured in this window).

Reply:
Oh my mistake, I misread the diff summary and didn't notice there was already SKIP logic here. :+1:

Reply:
This test actually runs only on the unix port because that's the only port with both lfs1 and lfs2 enabled.... might be good to refactor this test as a unittest (one day) so it can run on ports with lfs1 and/or lfs2.

But regardless of that, the skip logic should work for any configuration, as you noticed.